AmericanMedicalAssociation/ama-style-guide-2
# Adf 130 prototype search results header

**Jira Ticket**
- [ADF-130 : Prototype search results header](https://palantir.atlassian.net/browse/ADF-130)

## Description 

- Add Search results Header

## To Test

- Checkout the **ADF-130-prototype-search-results-header** branch and cd into styleguide, run gulp serve to spin up the pattern lab style guide.

- Verify that dr-finder-search-result-header folder and related files exist.

- Visit http://localhost:3000/?p=viewall-molecules-dr-finder-search-result-header and verify modal.

## Relevant Screenshots/GIFs
<img width="1353" alt="Screen Shot 2022-10-04 at 1 56 49 PM" src="https://user-images.githubusercontent.com/54925022/193915527-e171e0b2-5a53-4244-93b1-c4c3c6803a0a.png">
